README.md

```diff
@@ -81,6 +81,14 @@ java -jar composer-latest-version.jar \
 --verbose-output false
 ```
 
+### Download
+
+Composer [published to jcenter](https://jcenter.bintray.com/com/gojuno/composer) and follows [Semantic Versioning](http://semver.org/).
+
+You can download it in your CI scripts or store it in your version control system (not recommended).
+
+All the releases and changelogs can be found on [Releases Page](https://github.com/gojuno/composer/releases).
```

Comment:
Agree, but for now it is still possible to write some sort of custom `composer-launcher` and execute it instead of hosting fat JAR.